Converter/include/HierarchyBuilder.h

```diff
@@ -63,7 +63,7 @@ struct HierarchyBuilder{
 

```

Comment:
possibly add some mention of support for multiple files and explanation of reason

Reply:
Comments added